N/neper/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+using BinaryBuilder, Pkg
+
+name = "neper"
+version = v"4.5.0"
+
+sources = [
+    ArchiveSource("https://github.com/neperfepx/neper/archive/refs/tags/v$(version).tar.gz", "db80dd89e02207e9b056b05fb9fbe493199ce7c3736b2039104c595b4dcd02a9")
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/neper-*
+mkdir build && cd build
+CXXFLAGS="-lz"
+cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_CXX_FLAGS="${CXXFLAGS}" \
+      ../src
+make -j${nproc}
+make install
+"""
+
+function exclude(p)
+    if get(p.tags, "libc", nothing) == "musl"
+        # src/contrib/ut/src/ut_print/ut_print.c:1685:38: error: parameter 1 (‘beg_time’) has incomplete type
+        #  1685 | ut_print_elapsedtime (struct timeval beg_time, struct timeval end_time)
+        #       |                       ~~~~~~~~~~~~~~~^~~~~~~~
+        return true
+    elseif get(p.tags, "os", nothing) == "windows" && get(p.tags, "arch", nothing) == "i686"
+        # CMake Error at CMakeLists.txt:262 (install):
+        #  install FILES given no DESTINATION!
+        return true
+    elseif get(p.tags, "os", nothing) == "windows" && get(p.tags, "arch", nothing) == "x86_64"
+        # CMake Error at CMakeLists.txt:262 (install):
+        #  install FILES given no DESTINATION!
+        return true
+    elseif get(p.tags, "os", nothing) == "freebsd"
+        # CMake Error at CMakeLists.txt:262 (install):
+        #  install FILES given no DESTINATION!
+        return true
```

Comment:
This looks to be https://github.com/neperfepx/neper/blob/886d666e038507af5777dc12e7192b4f4d90ac7e/src/CMakeLists.txt#L256-L264